test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Connection/ReliableConnectionTests.cs

```diff
@@ -257,19 +274,19 @@ private DbConnection CreateTestConnection()
         /// Test ReliableConnectionHelper.GetDefaultDatabaseFilePath()
         /// </summary>
         [Fact]
-        public void TestGetDefaultDatabaseFilePath()
+        public async void TestGetDefaultDatabaseFilePath()
```

Comment:
should be `async Task` - this is recommended for async test return types. Void return can cause loss of failure / errors